docs/source/noise/index.rst

```diff
@@ -177,10 +177,12 @@ Noise types for each column
     - Decennial Census, ACS, CPS, WIC
     - Leave a field blank, choose the wrong option
     -
-  * - SSN
-    - W-2 and 1099, SSA
+  * - SSN or ITIN
+    - W-2 and 1099, 1040, SSA
     - "Borrowed" SSN, leave a field blank, write the wrong digits, make typos, make OCR errors
-    - Note that "borrowed" SSN only applies to the W-2 and 1099 dataset
+    - Note that "borrowed" SSN only applies to the W-2 and 1099 dataset.
+      ITIN is only reported in the 1040, and only when the simulant does not have an SSN.
+      In the 1040 form, this noise type functions the same for joint filer and dependent *n*
```

Comment:
```suggestion
      In the 1040 form, the same noise types apply to the SSN columns for the joint filer and dependents
```
As per my other comment, the column name should just be listed as "SSN" (not "SSN or ITIN"), so I don't think it's necessary to mention ITIN at all on the noise page. For the second sentence, here's a suggestion to make this row consistent with my other suggestions above.